op-challenger/game/fault/register_task.go

```diff
@@ -192,6 +192,42 @@ func NewAsteriscKonaRegisterTask(gameType faultTypes.GameType, cfg *config.Confi
 	}
 }
 
+func NewSuperAsteriscKonaRegisterTask(gameType faultTypes.GameType, cfg *config.Config, m caching.Metrics, serverExecutor vm.OracleServerExecutor, rootProvider super.RootProvider, syncValidator *super.SyncValidator) *RegisterTask {
+	stateConverter := asterisc.NewStateConverter(cfg.AsteriscKona)
+	return &RegisterTask{
+		gameType:               gameType,
+		syncValidator:          syncValidator,
+		skipPrestateValidation: gameType == faultTypes.SuperPermissionedGameType,
```

Comment:
nit: This isn't ever used for SuperPermissionedGameType so we could omit this line entirely.  Don't really mind either way though.

Reply:
Will keep for consistency for now :)